### PR TITLE
Add WrayJonathanM's pin to the map please!

### PR DESCRIPTION
**Pull request by WrayJonathanM**
Adding DFKI to the map if it's not already there

### CODE REVIEW

**Comment by crichID** [comment]
@WrayJonathanM I see you fixed the spelling mistake in the second file, but your tests will continue to fail because the first file is still part of your branch. I am going to delete the first file for you since it is a little tricky. Then the tests should pass.

**Comment by crichID** [comment]
@WrayJonathanM I deleted the file so the tests "should" pass now. In case you need to know in the future, here is a video that shows how you can access the same file and make some of the common corrections: https://vimeo.com/201011288

File: _pins/WrayJonathanM2.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: WrayJonathanM
+Latitude: 49.429580
```
Comment (crichID) [inline _pins/WrayJonathanM2.json]: @WrayJonathanM you will need to make Latitude and Longitude both a lowercase l in order for your tests to pass!

Comment (WrayJonathanM) [inline _pins/WrayJonathanM2.json]: thanks!

**Comment by brianamarie** [comment]
@WrayJonathanM I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**